exercises/practice/clock/.meta/config.json

```
@@ -1,11 +1,18 @@
 {
-  "authors": ["1ethanhanse"],
+  "authors": [
+    "1ethanhanse"
```

```suggestion
    "1ethanhansen"
```

might as well fix this while we're here